query/app.js

```diff
@@ -30,6 +30,26 @@ eventBus.subscribe('Volunteer Created', data => {
     volunteers.push(data);
 });
 
+//Photo API get method
+app.get("/photos", (req, res) => {
+    res.send(volunteers);
+});
+
+app.get("/photo/:photoId", (req, res) => {
+    const { photoId } = req.params;
+    let data;
+    photos.forEach(p => {
+        if(p.id === photoId)
+            data = p;
+    });
```

Comment:
Instead of iterating through the photos Array, checking for matching id, and changing data variable, you can utilize find function in Javascript. Something like...

```
const data = photos.find(photo => photo.id === photoId);
res.send(data)
```

.find stops the iteration of entire array when first entity of conditional is found, thus in terms of performance, faster than forEach  ( Unless there are multiple photos with same photoId present, which I don't think is the case here )

Reply:
Fixed